collections/_posts/2023-02-27-ConvNextV2.md

```diff
@@ -0,0 +1,150 @@
+    venue: arXiv
+pdf: "https://arxiv.org/abs/2301.00808"
+---
+
+# Notes
+
+* **Important : See this [review](https://creatis-myriad.github.io./2023/02/23/Conv-Next.html) to understand ConvNeXt V1**
+* Also you can refer to this [review](https://creatis-myriad.github.io/2022/08/31/MAE.html) about Masked Autoencoders (MAE)
+
+# Highlights
+
+* ConvNeXt V1 was focused on supervised learning only, however, similarly to transformers, convolution based models can benefit from self-supervised learning techniques (such as MAE)
+* Simply apply self-supervised learning methods (i.e. MAE) to ConvNeXt leads to sub optimal results
+* **This article proposes a fully convolutional masked autoencoder framework (MAE) and modifies the ConvNeXt architecture with Global Response Normalization (GRN) layers**
+
+
+
+# Fully convolutional masked autoencoder
+
+![](/collections/images/convnextV2/framework.jpg) 
+
+*Figure 1 : Fully convolutional masked autoencoder design*
+
+* They use a random masking strategy with a masking ratio of 0.6 
+
+* They use ConvNeXt as the encoder
+
+* Unlike transformers you can't simply remove masked patches from the image as the 2D structure of the image must be preserved for convolution
+
+* Also naive solutions such as replacing masked patches by *masked tokens* don't perform well in practice
+
+* The idea here is to see the masked image as a sparse data. Based on that, is natural to use sparse convolution that will operate only on the visible pixels
+  
+  | *Note 1 : if the center pixel of the convolution is masked then the convolution will not operate and just return a masked pixel*
+
+  |  *Note 2 :  the sparse convolution layers can be converted back to standard convolution at the fine-tuning stage without requiring additional handling*
+  
+* They tested several decoder architectures/depths but at the end they chose a simple ConvNeXt plain decoder 
+
+* Loss function is the mean squared error (MSE) computed only on the masked patches between reconstructed and target images
+  
+  ## Evaluation of FCMAE
+
+* They pre-train and fine-tune on ImageNet-1K for 800 and 100 epochs respectively and report top-1 accuracy
+
+* Ablation study is done to justify design choice
+
+  ![](/collections/images/convnextV2/sparse_conv.jpg)
+
+  ![](/collections/images/convnextV2/ablation1.jpg)
+
+  *Figure 2 : Results of the FCMAE's ablation study*
+
+* The also compare the self-supervised approach to fully supervised learning
+
+  <img src="/collections/images/convnextV2/result1.jpg" style="zoom:70%;" />
+
+  *Figure 3 : Comparison with fully supervised approach*
+
+* **They perform better than the fully supervised setup on 100 epochs but are still worse than the  orginal ConvNeXt V1 baseline train on 300 epochs**
+
+  | *This is in contrast to the recent success of masked image modeling using transformer-based models [..] where the pre-trained models significantly outperform the supervised counterparts.*
```

Comment:
For quotes, I suggest using the built-in quote formatting offered by markdown. In that case, you can even get rid of the italic formatting (*), because quotes are by default in italic.
```suggestion
> This is in contrast to the recent success of masked image modeling using transformer-based models [..] where the pre-trained models significantly outperform the supervised counterparts.
```